test/powershell/engine/Module/NewModuleManifest.Tests.ps1

```diff
@@ -20,6 +20,26 @@ Describe "New-ModuleManifest tests" -tags "CI" {
         $module.PrivateData.PSData.ProjectUri | Should BeExactly $absoluteUri
     }
 
+    function TestNewModuleManifestEncoding {
+        param ([byte[]]$expected)
+        New-ModuleManifest -Path $testModulePath
+        (Get-Content -Encoding Byte -Path $testModulePath -TotalCount $expected.Length) -join ',' | Should Be ($expected -join ',')
+    }
+
+    It "Verify module manifest encoding on Windows " -Skip:(-not $IsWindows) {
```

Comment:
Could you just calculate the expected value in a `beforeall` then have one test for all platforms?